kapitan/cli.py

```diff
@@ -64,7 +64,7 @@ def main():
                                 help='set search path, default is "."')
     compile_parser.add_argument('--verbose', '-v', help='set verbose mode',
                                 action='store_true', default=False)
-    compile_parser.add_argument('--no-prune', help='do not prune jsonnet output',
```

Comment:
Maybe we could keep it for a while and add a warning that it will be removed in the next version?